book/website/_toc.yml

```diff
@@ -653,6 +653,8 @@ parts:
                 file: community-handbook/consistency/consistency-structure
               - title: Language
                 file: community-handbook/consistency/consistency-language
+          - title: Build the Turing Way Book locally
```

Comment:
Just a note that this title doesn't match the title of the Markdown file

Reply:
good spot! 🧐